amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -18,7 +18,7 @@ enum EmailPermissions {
 }
 
 input ProfileMetadataInput {
-  emailPermissions: [EmailPermissions!]!
+  emailPermissions: [String!]!
```

Comment:
With `[EmailPermissions!]!` I was intending a type of `("isHuman" | "sendNotifications")[]`, but instead it generates `EmailPermissions | Array<EmailPermisisons>` which is not the same thing. 

Any one know if you can get the intended effect in GraphQL? Or I just need to leave it as a general "String" array?

Also, anyone know why codegen turns a type of `[x]!` into `x | x[]` ? Don't see how this leads to the implication that there won't be an array.